### ISSUE TITLE
array_combine(): Argument #1 ($keys) and argument #2 ($values) must have the same number of elements

### ISSUE THREAD

**Issue by FractalizeR** [PromPHP/prometheus_client_php#147]
If you first send a metric with two labels and later remove one of the labels, you get an exception in \Prometheus\RenderTextFormat::escapeAllLabels because two label arrays cannot be array_combine()'ed. You have to flush metric storage to make it working again.

**Comment by stevenbrookes** [comment]
Yes I just got hit by the same problem.  In my case redis storage, removing the keys of the modified metrics fixed the issue, but a pain to deal with.
